pkg/cleaner/azure/cleaner.go

```diff
@@ -4,6 +4,7 @@ import (
 	"context"
 	"strings"
 
+	"github.com/Azure/azure-sdk-for-go/services/dns/mgmt/2017-10-01/dns"
```

Comment:
Why is this import so much "older" than the others? Or is there simply no newer version?

Reply:
That's the newest beside `dns/mgmt/2018-03-01-preview/dns` where I am not sure what preview stands for, alpha/beta version?
So I rather go with the sable version.

Reply:
Ack, makes sense.